R/bayesianmetaanalysiscommon.R

```diff
@@ -1185,51 +1220,54 @@
 .bmaForestPlot <- function(jaspResults, dataset, options, ready, .bmaDependencies) {
   forestContainer <- createJaspContainer(title = gettext("Forest Plot"))
   forestContainer$dependOn(c(.bmaDependencies, "studyLabels"))
+  forestContainer$position <- 6
   jaspResults[["forestContainer"]] <- forestContainer
-  jaspResults[["forestContainer"]]$position <- 6
   
   # Get studylabels
   if(options[["studyLabels"]] != ""){
-    studyLabels <- as.character(dataset[, .v(options[["studyLabels"]])])
+    studyLabels <- as.character(dataset[, options[["studyLabels"]]])
   } else {
     studyLabels <- paste(gettext("Study"), 1:nrow(dataset))
   }
   
-  # Scale the height and width of the plot
-  height <- 100 + nrow(dataset) * 30
-  width  <- 500 + (nchar(max(studyLabels)) * 5)
-  
-  # title of plot based on observed/estimated 
-  if(options$forestPlot == "plotForestBoth"){
-    title  <- gettext("Observed and estimated study effects")
-    height <- height + 50 + nrow(dataset) * 30
-  } else  if(options$forestPlot == "plotForestEstimated"){
-    title <- gettext("Estimated study effects")
-  } else  if(options$forestPlot == "plotForestObserved" || options$modelSpecification == "FE"){
-    title <- gettext("Observed study effects")
-  }
-  
   # Check if ready    
   if(!ready){
     return()
   } 
   
+  # Scale the height and width of the plot
+  heightCum <- 100 + nrow(dataset) * 30 
```

Comment:
I personally prefer to call variable names like `heightCumulative` or `totalHeight` - it's only a couple of extra key strokes and adds clarity to the code. But it's your choice of course whether to change it...

Reply:
I changed it to `heightCumulative` :)